src/components/leaderboard/GlobalStats/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { mutedTextColorStyles } from '@/components/content-staking/utils/commonStyles'
+import { isTouchDevice } from '@/utils/device'
+import { IoPeople } from 'react-icons/io5'
+import { cx } from '../../../utils/class-names'
+import LeaderboardTable from '../MyStats/LeaderboardTable'
+import GlobalStatsDashboard from './GlobalStatsDashboard'
+
+const customColumnsClassNames = ['md:w-[9.8%]', undefined, 'md:w-[30%]']
+
+const GlobalStats = () => {
+  return (
+    <div className='flex w-full flex-col gap-5'>
+      <div className='grid grid-cols-2 flex-col gap-4 md:grid-cols-3'>
+        <div
+          className={cx(
+            'col-[span_2] flex gap-4 rounded-[20px] border border-[#6366F1]/20 bg-[#0053FF]/10 md:col-auto md:row-[span_2]',
+            'items-center p-4 md:flex-col'
+          )}
+        >
+          <div
+            className='flex items-center justify-center rounded-full'
+            style={{
+              background: 'white',
+              width: isTouchDevice() ? '70px' : '88px',
+              height: isTouchDevice() ? '70px' : '88px',
```

Comment:
make sense to use one const for it like

const sizeStyle = isTouchDevice() ? '70px' : '88px'